internals/secrethub/generate_test.go

```diff
@@ -184,17 +221,16 @@ func TestGenerateSecretCommand_run(t *testing.T) {
 				}, tc.clientCreationErr
 			}
 
-			io := fakeui.NewIO(t)
-			tc.cmd.io = io
-
 			// Act
 			err := tc.cmd.run()
+			resClip, _ := tc.cmd.clipper.ReadAll()
```

Comment:
After this line you could call `assert.OK` on the error value, to make sure it is always nil.

Reply:
Good to know. 
Done